README.md

```diff
@@ -1 +1,105 @@
-# node-modules-json
\ No newline at end of file
+# Importing JSON files
+
+## Overview
+
+This lesson will cover how and why you would import JSON files.
+
+## Objectives
+
+1. Describe how to import JSON files using provided example: configurations, data, settings
+1. Describe why you'd need to have JSON files
+
+## Reading JSON Files
+
+JSON stands for JavaScript Object Notation. It's a widely use data format. When it comes to Node applications, developers like to use JSON file to store various data such as:
+
+* Configurations
+* Seed data (initial data to populate the development or test database)
+* Settings
+* Plugins for frameworks
+
+
+Let's implement code which reads a JSON file and parses the file data into JavaScript object (`read.js`):
+
+```js
+var fs = require('fs')
+fs.readFile('./customers.json', {encoding: 'utf8'}, function(error, jsonString){
+  try {
+    var data = JSON.parse(jsonString)
+  } catch(e) {
+    console.log(e)
+  }
+  console.log(data.length, data[2].email)
+})
+```
+
+The fact that we can access property `data[2].email` tells us that `data` is a JavaScript/Node object and not a JSON string.
+
+All the code above can be replaced with one line where we use `require()`—`req.js`:
+
+```js
+var data = require('./customers.json')
+console.log(data.length, data[2].email)
+```
```

Comment:
tell us how we have access to `.length` , is it cause the JSON is stored in array? explain how we have access to `.email` because the JSON contains a "key" called "email"